scripts/curve/lido_gauge_incentives.py

```diff
@@ -22,7 +28,7 @@ def main(amount=0, epoch=MONTH, use_wsteth=True):
     """
     safe = GreatApeSafe(TROPS)
     gauge = safe.contract(GAUGE, Interface=interface.ILiquidityGaugeV6)
-    amount = int(float(amount) * 1e18)
+    amount = int(Decimal(amount) * 1e18)
```

Comment:
did you test this? i dont think you can multiply a `Decimal` with a non `Decimal` type:
```
>>> int(Decimal(4.22)*1e18)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for *: 'decimal.Decimal' and 'float'
```